### DISCUSSION TITLE
Wifi Issues Help

### DISCUSSION THREAD

**Discussion by Mark18PLL** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#804] [General]
Ok so I have hooked everything up and it all seems to connect and work perfect for a few minutes or so and then just hangs.  I have checked the signal which is pretty good and the pump is only about 5 metres away from the access point in direct sight.   I have gone through the setup instructions and noticed the following info but I dont know how to implement it, is it something that can be done over wifi if I have a signal for a few minutes?  I purchased the module complete so I did not make it up myself of programme it. 

**Device hangs if WiFi is bad
If not connected to WiFi the device will go into Access point mode and wait for you to enter your credentials. To avoid this,
edit file "config.h" and set bool enableAp = false;**

Thanks 

Mark

**Comment by visualapproach** [comment]
> Device hangs if WiFi is bad

That should not be the case anymore. I should update the manual if that's where you found it. Maybe true if you have an old version of FW. 

If it's the access point "layzspaXXXXX" you can't reach it may also be a later version with a bug that would make it difficult to connect.
(Not in the latest version though)

Do you know what FW version you have? You can upload new FW yourself by following the manual, if that's the problem.

**Reply by Mark18PLL** [reply]
Just checked and the firmware is 2024-07-23-1743

**Reply by visualapproach** [reply]
Ok, so that should not hang due to wifi. You could see if the development branch is better, but I', not sure. What does the bootlog.txt say?

**Reply by Mark18PLL** [reply]
This is from the bootleg.txt 

{"BOOTINFO":"Power On 2024-10-05 13:30:12"}
{"BOOTINFO":"Power On 2024-10-05 15:46:12"}
{"BOOTINFO":"Power On 2024-10-05 16:02:53"}
{"BOOTINFO":"Power On 2024-10-05 16:12:28"}
{"BOOTINFO":"Software/System restart 2024-10-05 16:54:16"}
{"BOOTINFO":"Power On 2024-10-05 16:58:20"}
{"BOOTINFO":"Power On 2024-10-05 17:14:23"}
{"BOOTINFO":"Power On 2024-10-05 17:48:33"}
{"BOOTINFO":"Power On 2024-10-05 17:50:08"}
{"BOOTINFO":"Power On 2024-10-05 17:56:41"}
{"BOOTINFO":"Power On 2024-10-05 18:00:16"}
{"BOOTINFO":"External System 2024-10-05 18:04:21"}
{"BOOTINFO":"External System 2024-10-05 18:07:58"}
{"BOOTINFO":"Power On 2024-10-05 18:11:41"}
{"BOOTINFO":"Power On 2024-10-05 18:21:59"}
{"BOOTINFO":"Software/System restart 2024-10-05 18:27:24"}
{"BOOTINFO":"Power On 2024-10-05 18:31:19"}
{"BOOTINFO":"External System 2024-10-05 18:32:56"}
{"BOOTINFO":"External System 2024-10-05 18:35:02"}
{"BOOTINFO":"External System 2024-10-05 18:36:12"}
{"BOOTINFO":"External System 2024-10-05 18:37:13"}
{"BOOTINFO":"External System 2024-10-05 18:37:42"}
{"BOOTINFO":"Power On 2024-10-05 18:40:54"}
{"BOOTINFO":"External System 2024-10-05 18:42:06"}
{"BOOTINFO":"External System 2024-10-05 18:42:37"}
{"BOOTINFO":"Power On 2024-10-05 18:45:08"}
{"BOOTINFO":"Software/System restart 2024-10-05 18:48:36"}
{"BOOTINFO":"Power On 2024-10-05 18:50:16"}
{"BOOTINFO":"Hardware Watchdog 2024-10-05 18:58:11"}
{"BOOTINFO":"Software/System restart 2024-10-05 19:07:03"}
{"BOOTINFO":"Power On 2024-10-05 19:10:09"}
{"BOOTINFO":"Software/System restart 2024-10-05 19:24:13"}
{"BOOTINFO":"Power On 2024-10-05 19:28:22"}
{"BOOTINFO":"Power On 2024-10-05 19:37:17"}
{"BOOTINFO":"Power On 2024-10-05 19:58:57"}
{"BOOTINFO":"Exception 2024-10-06 05:48:52"}
{"BOOTINFO":"Software/System restart 2024-10-06 07:11:04"}
{"BOOTINFO":"Software/System restart 2024-10-06 07:11:44"}
{"BOOTINFO":"Exception 2024-10-06 08:06:42"}
{"BOOTINFO":"Power On 2024-10-06 08:59:10"}
{"BOOTINFO":"Power On 2024-10-06 10:02:57"}
{"BOOTINFO":"Software/System restart 2024-10-06 10:12:05"}

**Comment by Mark18PLL** [comment]
I think I might have found the solution, I understand networking a little but im certainly no expert.  Instead of using the lazyspa.local if I login using the ip address it seems to work for now and it hasn't disconnected. 
thanks 
Mark

**Reply by visualapproach** [reply]
Ok hope for the best. Maybe a router restart will help. So the hanging was only that you couldn't load the web pages? The display buttons still worked?

**Reply by Mark18PLL** [reply]
Yes display buttons work fine, just the web page hanging. 
Thanks

**Reply by visualapproach** [reply]
Thanks @Mark18PLL for the beer! Have a good day!
